Formula/massren.rb

```diff
@@ -18,6 +18,7 @@ class Massren < Formula
 
   def install
     ENV["GOPATH"] = buildpath
```

Comment:
Is this line not redundant with `GO111MODULE` set?

Reply:
No, this env just restores Go 1.15 behaviour.

It's really actually the same as `GO111MODULE=off` here, except it will use modules (i.e. switch back to 1.16's new default of `on`) should they be added in an update and someone forgets to remove the env.